### ISSUE TITLE
Error  in SplineConv

### ISSUE THREAD

**Issue by DimaZhu** [rusty1s/pytorch_geometric#15]
Can't run basic example cora.py. Error occurred: 
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-12-0d09e6efa5bf> in <module>()
     54 
     55 for epoch in range(1, 201):
---> 56     train()
     57     log = 'Epoch: {:03d}, Train: {:.4f}, Test: {:.4f}'
     58     print(log.format(epoch, *test()))

<ipython-input-12-0d09e6efa5bf> in train()
     39     model.train()
     40     optimizer.zero_grad()
---> 41     F.nll_loss(model()[data.train_mask], data.y[data.train_mask]).backward()
     42     optimizer.step()
     43 

/usr/local/lib/python3.5/dist-packages/torch/nn/modules/module.py in __call__(self, *input, **kwargs)
    489             result = self._slow_forward(*input, **kwargs)
    490         else:
--> 491             result = self.forward(*input, **kwargs)
    492         for hook in self._forward_hooks.values():
    493             hook_result = hook(self, input, result)

<ipython-input-12-0d09e6efa5bf> in forward(self)
     25         x, edge_index, edge_attr = data.x, data.edge_index, data.edge_attr
     26         x = F.dropout(x, training=self.training)
---> 27         x = F.elu(self.conv1(x, edge_index, edge_attr))
     28         x = F.dropout(x, training=self.training)
     29         x = self.conv2(x, edge_index, edge_attr)

/usr/local/lib/python3.5/dist-packages/torch/nn/modules/module.py in __call__(self, *input, **kwargs)
    489             result = self._slow_forward(*input, **kwargs)
    490         else:
--> 491             result = self.forward(*input, **kwargs)
    492         for hook in self._forward_hooks.values():
    493             hook_result = hook(self, input, result)

/usr/local/lib/python3.5/dist-packages/torch_geometric/nn/conv/spline_conv.py in forward(self, x, edge_index, pseudo)
     81                           self._buffers['kernel_size'],
     82                           self._buffers['is_open_spline'], self.degree,
---> 83                           self.norm, self.root, self.bias)
     84 
     85     def __repr__(self):

/usr/local/lib/python3.5/dist-packages/torch_spline_conv/conv.py in apply(src, edge_index, pseudo, weight, kernel_size, is_open_spline, degree, norm, root_weight, bias)
     53 
     54         src = src.unsqueeze(-1) if src.dim() == 1 else src
---> 55         pseudo = pseudo.unsqueeze(-1) if pseudo.dim() == 1 else pseudo
     56 
     57         row, col = edge_index

AttributeError: 'NoneType' object has no attribute 'dim

When changing SplineConv to GCNConv everything works.

**Comment by rusty1s** [comment]
It seems that `edge_attr` on line 25 is `None`. Can you confirm this?

**Comment by DimaZhu** [comment]
Yes, it's `None`

**Comment by rusty1s** [comment]
Did you modify the Cora example? The `TargetIndegree` transform should add edge attributes to the Cora graph:

```
data = Planetoid(path, dataset, T.TargetIndegree())[0]
```

**Comment by DimaZhu** [comment]
OK, solved it. What's was wrong in my case. I had to run jupyter in docker. So I wrote in Dockfile instructions from docs. But there was error while import pytorch_unique "No module named 'unique_cuda". When I tried to install this package in already running container everything was fine. But there was error from above. 
Now I executed all commands from docs in already running container and everything is OK. Maybe order of packages matters or something else. But it works.
By the way, you did great work! Thanks! It's really pleased to see paper with source code.
